Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/Bindings/ClickBinding.cs

```diff
@@ -14,7 +14,7 @@ public class ClickBinding
     /// <example>When user clicks on "Test" button</example>
     [Given(@"user clicked on ""(.+?)""")]
     [When(@"user clicks on ""(.+?)""")]
-    public async Task ClickOnElement(IWebElementWrapper element)
+    public async Task ClickOnElement(WebElementWrapper element)
```

Comment:
Why change interface to concrete implementation?

Reply:
The framework is intended to be simple and light. So without the abstract layer, it will be more straightforward.

Reply:
And how do you intend to override default wrappers that you have in new projects in case of compatibility issues?

Reply:
To create a new one or edit existing ones. Since we don't have a NuGet package the solution will be simple cloned and customized.